libgobuster/statehelpers.go

```diff
@@ -170,6 +171,22 @@ func ValidateDirModeState(
 		}
 	}
 
+	// Read the user agents file
+	if errorList.ErrorOrNil() == nil && s.RandomUserAgent {
+		b, err := ioutil.ReadFile(s.UserAgentsFile)
+		if err != nil {
+			errorList = multierror.Append(errorList, fmt.Errorf("[!] Error when reading the User-Agents file"))
+		}
+		lines := strings.Split(string(b), "\n")
+		var uas []string
+		for _, line := range lines {
+			if !strings.HasPrefix(line, "#") && len(line) > 0 {
```

Comment:
I would probably call `strings.TrimSpace` on the line before checking for the prefix:

* https://golang.org/pkg/strings/#TrimSpace